sympy/core/tests/test_power.py

```diff
@@ -579,3 +579,7 @@ def _(a, b):
     assert power(a, b) == NewPow(a, b)
     assert power(b, a) == NewPow(b, a)
     assert power(b, b) == NewPow(b, b)
+
+def test_issue_19572():
+    assert 0 ** -oo is zoo
+    assert power(0, -oo) is zoo
```

Comment:
Can you add this to `test_zero` above rather than a new test function? You can include the url for the issue in a comment.

Reply:
Yes sure